src/uk/ac/open/audio/streaming/PlayerProgress.java

```diff
@@ -189,6 +194,8 @@ private synchronized boolean setState(State newState)
 		{
 			playMilliseconds=0;
 			playPercentage=0.0;
+			this.pausetime=0;
+			TabApplet.pauseFlag=false;
```

Comment:
I'm really not sure pauseFlag as a static value in TabApplet is a good idea - this might work badly with multithreading and other aspects. Is it not possible to think of somewhere sensible to store it, like maybe it should be stored inside this PlayerProgress object even? Not sure.